non-free.md

```diff
@@ -51,6 +51,9 @@
 
 - [Elastix](http://www.elastix.org) `⊘ Proprietary` - Unified communications server software based on 3CX.
 
+## Knowledge Management Tools
+
+- [Wiznote](https://www.wiz.cn/docker) `⊘ Proprietary` - Manage your knowledge in documents with folders, markdown, full text search and webpage collection support. ([Demo](https://www.wiz.cn/), [Clients](https://www.wiz.cn/download.html)) `⚠` `English/Chinese`
```

Comment:
The language tag referes to the software server side rather than the language of written aspects of the software. Please change `English/Chinese` to `Docker`

Reply:
They are fixed 😁